src/me/corriekay/pokegoutil/utils/Utilities.java

```diff
@@ -45,13 +45,14 @@ public static boolean isEven(long i) {
 
     /**
      * Takes to numbers and creates a decimal percentage of it (like 0.7542).
```

Comment:
Doen's about this commit but maybe there is a typo here? "Takes TWO..." instead of "Takes to"?